CommandLine/Option.swift

```diff
@@ -357,25 +348,7 @@ public class EnumOption<T:RawRepresentable where T.RawValue == String>: Option {
    * of Xcode 7 beta 2.
    */
   
-  private override init(_ shortFlag: String?, _ longFlag: String?, _ required: Bool, _ helpMessage: String) {
-    super.init(shortFlag, longFlag, required, helpMessage)
-  }
-  
-  /** Initializes a new Option that has both long and short flags. */
-  public convenience init(shortFlag: String, longFlag: String, required: Bool = false, helpMessage: String) {
-    self.init(shortFlag as String?, longFlag, required, helpMessage)
-  }
-  
-  /** Initializes a new Option that has only a short flag. */
-  public convenience init(shortFlag: String, required: Bool = false, helpMessage: String) {
-    self.init(shortFlag as String?, nil, required, helpMessage)
-  }
-  
-  /** Initializes a new Option that has only a long flag. */
-  public convenience init(longFlag: String, required: Bool = false, helpMessage: String) {
-    self.init(nil, longFlag as String?, required, helpMessage)
-  }
-  
```

Comment:
Since you removed these initializers, you should also take the comment about them above.